openshift_sti_images/python.adoc

```diff
@@ -0,0 +1,191 @@
+= Python
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+The OpenShift https://github.com/openshift/sti-python/tree/master/3.3[Python] image repository contains the sources and Dockerfiles for building various versions of Python platform images. The resulting images can be run either by Docker or using https://github.com/openshift/source-to-image[STI].
+
+== Versions
+Currently, OpenShift only supports version https://github.com/openshift/sti-python/tree/master/3.3[3.3] of Python.
+
+== Base Images
+
+* RHEL-7
+* CentOS-7
+
+[NOTE]
+====
+In order to perform build or test action on a RHEL-7 based Python image, you need to run the build on properly subscribed RHEL-7 machine.
+====
+
+== Installation
+To build a Python image, choose between a RHEL-7 or CentOS-7 base image:
+
+=== RHEL-7 base image
+
+----
+$ git clone https://github.com/openshift/sti-python.git
+$ cd sti-python
+$ make build TARGET=rhel7 VERSION=3.3
+----
+
+=== CentOS-7 Base Image
+
+----
+$ git clone https://github.com/openshift/sti-python.git
+$ cd sti-python
+$ make build VERSION=3.3
+----
+
```

Comment:
"or just pull it from docker hub......"  (again all our docs should say this for the centos images)